### PR TITLE
Refactor iframe resizing

### PR DESCRIPTION
**Pull request by perprogramming**
- Presentation belongs to css
- Progressive enhancement: Do not define onlick handles in html attributes
- Use jquery as it is already loaded
- document.ready + binding of click handlers prevents polluting global scope

@soundyogi What'ya think?

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,32 +1,23 @@
...
-
-}
\ No newline at end of file
+var $ = require('jquery');
+require('lity');
+
+$(document).ready(function () {
+  var lastbreakPoint;
+
+  $('.resize-iframe-toggle').click(function () {
```
Comment (perprogramming) [inline js/index.js:14]: @soundyogi was unsure how to call the classes or wheter to reuse ck-labsizer__item for this.. I once used the convention to prefix classes that bind js code with "fx-"..

File: scss/lab.scss [hunk 2]
```diff
@@ -21,6 +21,18 @@ iframe {
   transition: 1s width linear;
 }
 
+iframe.small {
```
Comment (perprogramming) [inline scss/lab.scss:25]: @soundyogi do we need a block identifier here? e.g. ck-pattern-frame?

Comment (soundyogi) [inline scss/lab.scss:25]: Hm yes but:
It should work by thursday anything else can be refactored.
(I think I will get to it wednesday)

Id say leave it for now - refactor later

File: js/index.js [hunk 3]
```diff
@@ -1,32 +1,23 @@
...
+      if (lastbreakPoint) {
+        $(this).removeClass(lastbreakPoint);
+      }
+      $(this).addClass(breakpoint);
+    })
+    lastbreakPoint = breakpoint;
+  }).first().click();
+
+  $(window).load(function() {
+    $('iframe').each(function () {
```
Comment (soundyogi) [inline js/index.js]: Its fine but the height resizing has to take place after every breakpoint change aswell

**Comment by soundyogi** [comment]
> Presentation belongs to css 

Definetly

> Progressive enhancement: Do not define onlick handles in html attributes

Totally

> Use jquery as it is already loaded

100%

> document.ready + binding of click handlers prevents polluting global scope

Yup

**Pull request merged**